### ISSUE TITLE
[Question] How do you remove aria-hidden=true on navigation?

### ISSUE THREAD

**Issue by iaindarkflare** [ResponsiveBP/Responsive#67]
Hi James,

I'm changing an existing site over to v4 from v3 and noticed that I needed to tweak some css (adding display:block to #navigation) to show my dropdown nav on wider displays. I'm using a dropdown to toggle the nav on mobile.

When I was having a look in Developer Tools to suss this out I noticed that the navigation is set to aria-hidden=true on wider displays as I'm forcing it to be expanded. The role changes as expected on mobile.

Is there something I'm missing to have this change? Don't suppose it's a big issue, but wanted to ask as it looks like responsive sets  `display: none !important;` on items with `[aria-hidden="true"]` and I notice that on the docs site you have the main nav set as aria-hidden=false on wider displays as you'd expect.

Are there any drawbacks to leaving it as  `[aria-hidden="false"]` and forcing it's appearance with `display: block!important;`?

Cheers,
Iain

**Comment by JimBobSquarePants** [comment]
Hi Ian,

>Are there any drawbacks to leaving it as [aria-hidden="false"] and forcing it's appearance with display: block!important;?

I take it you mean `[aria-hidden="true"]`? If so then yes. From an accessibility point of view you don't want the two to be mixed up as you prevent screen readers from reading the content. You'll notice also that the `tabindex` attribute changes also which will affect keyboard accessibility.

I use a little snippet of JavaScript code in the documentation to ensure that those attributes are changed.
https://github.com/ResponsiveBP/Responsive/blob/gh-pages/assets/js/docs.js#L6


I use this css to then tweak the navigation styles in a media query. Really I should probably use the attribute selector though.
https://github.com/ResponsiveBP/Responsive/blob/gh-pages/assets/css/docs.css#L82

Hope that helps.

James

**Comment by iaindarkflare** [comment]
Hi James,

Yes I did mean `[aria-hidden="true"]`, sorry about that!

Do you mind if I use the JS snippet myself?

As for the CSS I was previously using…
    #navigation.collapse {
            height: auto;
            overflow: visible;
    }

…but noticed that wasn't working on the new version, so was trying to suss out why I needed to use `display: block` as well this time around.

If I can use the JS above that shouldn't be needed as the role will change, but I will look to used the attribute selector.

Cheers,
Iain

**Comment by JimBobSquarePants** [comment]
Yeah fire away chum, use anything you like :smile:

**Comment by iaindarkflare** [comment]
Awesome, thanks again for your help!
